velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -854,5 +855,62 @@ TEST_F(Re2FunctionsTest, regexExtractAllBadArgs) {
   EXPECT_THROW(eval("123", "[a-z]+", 1), VeloxException);
 }
 
+TEST_F(Re2FunctionsTest, tryException) {
```

Comment:
It looks like this test verifies cases where all rows fail. Would you add some test cases when only some rows fail?